apps/web/src/ee/billing/hooks/useSubscription.ts

```diff
@@ -36,6 +36,16 @@ export const useSubscription = () => {
           daysTotal: 0,
         },
       },
+      select: (data) => {
+        return {
+          ...data,
+          events: {
+            ...data.events,
+            // if included is null, customer doesn't have a valid metered subcription, default to 0
+            included: data.events.included ?? 0,
```

Comment:
When implement steps to ensure that all users have an active subscription, then we can remove this.